examples/arithmetic/data/sample/testdata.yaml

```diff
@@ -2,4 +2,3 @@ on_reject: continue
 range: 0 0
 accept_score: 0
 grader_flags: first_error
-input_validator_flags: ab=1e9 c=1e9
```

Comment:
This seems wrong, I guess you canceled running generator.sh partway through?

Reply:
I may have, or a bug, will review

Reply:
Fixed, was from cancelling